# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bmon/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- bmon-2.1.0/src/out_xml_event.c.vanilla	2005-10-20 15:25:06.000000000 +0200
+++ bmon-2.1.0/src/out_xml_event.c	2005-10-20 15:25:16.000000000 +0200
@@ -127,7 +127,7 @@
 	.om_draw = xml_event_draw,
 	.om_set_opts = xml_event_set_opts,
 	.om_probe = xml_event_probe,
-	.om_shutdown xml_event_shutdown,
+	.om_shutdown = xml_event_shutdown,
 };
 
 static void __init xml_event_init(void)
--- bmon-2.1.0/src/out_audio.c.vanilla	2005-10-20 15:25:26.000000000 +0200
+++ bmon-2.1.0/src/out_audio.c	2005-10-20 15:25:31.000000000 +0200
@@ -141,7 +141,7 @@
 	.om_draw = audio_draw,
 	.om_set_opts = audio_set_opts,
 	.om_probe = audio_probe,
-	.om_shutdown audio_shutdown,
+	.om_shutdown = audio_shutdown,
 };
 
 static void __init audio_init(void)
